pom.xml

```diff
@@ -0,0 +1,212 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>Kickstarter</groupId>
+    <artifactId>Kickstarter</artifactId>
+    <version>0.1</version>
```

Comment:
Google for: semantic versioning. Most of the companies I've been to use x.x.x or x.x.x_x (or see how your libs are named)